src/components/Tokens/TokenDetails/TokenDetail.tsx

```diff
@@ -225,18 +309,7 @@ export default function LoadedTokenDetail({ address }: { address: string }) {
           </MissingChartData>
         </ChartHeader>
         <MissingData>
-          <AboutSection>
-            <AboutHeader>
-              <Trans>About</Trans>
-            </AboutHeader>
-            <NoInfoAvailable>
-              <Trans>No token information available</Trans>
-            </NoInfoAvailable>
-            <ResourcesContainer>
-              <Resource name={'Etherscan'} link={`https://etherscan.io/address/${address}`} />
-              <Resource name={'Protocol Info'} link={`https://info.uniswap.org/#/tokens/${address}`} />
-            </ResourcesContainer>
-          </AboutSection>
+          <AboutSection address={address} tokenDetailData={tokenDetailData as TokenDetailData} />
```

Comment:
why do you need to cast?

Reply:
it has more props than we need

Reply:
visual studio complains when we dont cast

Reply:
you cna make a new object without the extra props